openlibrary/templates/type/edition/view.html

```diff
@@ -80,7 +80,9 @@
 $ book_title = edition.get('title', '') or (work.title if work else '')
 $ work_title = work.title if work else edition.get('title', '')
 $ book_subtitle = edition.get('subtitle', '')
-$ title_suffix = cond(edition.get('publish_date'), u"({0} edition)".format(edition.get('publish_date')), "(edition)")
+$ authors = work.get_authors() if work else edition.get('authors', {})
```

Comment:
```suggestion
$ authors = work.get_authors() if work else edition.get_authors()
```
`Edition`s also have a `get_authors()` method.